Common/src/CConfig.cpp

```diff
@@ -6603,6 +6603,18 @@ void CConfig::SetOutput(SU2_COMPONENT val_software, unsigned short val_izone) {
           cout << "Venkatakrishnan slope-limiting method, with constant: " << Venkat_LimiterCoeff << ".\n";
           cout << "The reference element size is: " << RefElemLength << ". " << endl;
           break;
+        case LIMITER::NISHIKAWAR3:
+          cout << "NISHIKAWAR3 slope-limiting method, with constant: " << Venkat_LimiterCoeff << ".\n";
```

Comment:
Same here (apply to R4 and R5 too please)
```suggestion
          cout << "Nishikawa's R3 slope-limiting method, with constant: " << Venkat_LimiterCoeff << ".\n";
```